packages/react-composites/src/composites/CallComposite/Strings.tsx

```diff
@@ -314,6 +314,26 @@ export interface CallCompositeStrings {
    * More details text of the page shown to the user when the user attempts to join a room that cannot be found.
    */
   roomNotFoundDetails?: string;
+  /* @conditional-compile-remove(video-background-effects) */
+  /**
+   * Label for the blur video background effect item
+   */
+  blurBackgroundEffect?: string;
+  /* @conditional-compile-remove(video-background-effects) */
+  /**
+   * Tooltip text for the blur video background effect item
+   */
+  blurBackgroundTooltip?: string;
+  /* @conditional-compile-remove(video-background-effects) */
+  /**
+   * Label for the remove video background effect item
+   */
+  removeBackgroundEffect?: string;
```

Comment:
is removeBackgroundEffectButtonLabel more descriptive?

Reply:
done